tools/openocd/adafruit-esp.cfg

```diff
@@ -0,0 +1,13 @@
+
```

Comment:
Why is adafruit in the filename?

Reply:
The interface we are using is the adafruit breakout board. A different sourcing of the ft2232h chip could have a different VID. In the future, if we have the same VID on the chip used on the Hiveboard, we could rename it to hiveboard-esp.cfg.

Reply:
Thanks for the clarification!